Dockerfile

```diff
@@ -49,19 +41,14 @@ COPY entrypoint.sh scripts/receive_imap.sh scripts/update-repositories.sh script
 COPY crontab ${REDMINE_LOCAL_PATH}/
 
 WORKDIR $REDMINE_PATH
-# OLD PATCH https://www.redmine.org/attachments/download/20934/0001-Allow-the-current-user-to-log-time-for-other-users.patch
-# NEW PATCH https://www.redmine.org/attachments/download/22481/allow_watchers_and_contributers_access_to_issues_4.0.2.patch
-# See also https://www.redmine.org/attachments/download/7349/subprojects_query_filter_fix.patch
-ADD https://www.redmine.org/attachments/download/22481/allow_watchers_and_contributers_access_to_issues_4.0.2.patch \
+
+ADD patches/allow_watchers_and_contributers_access_to_issues_4.2.2.patch \
     patches/redmine_3_6_log_time_for_others.patch \
```

Comment:
This patch is ADDed in the Dockerfile, but not applied. However, functionality seems to work as expected in test instance.